plugin.xml

```diff
@@ -63,7 +63,7 @@
     <!--========== Android ==========-->
     <platform name="android">
         <config-file target="res/xml/config.xml" parent="/*">
-            <feature name="BlinkUpPlugin">
+            <feature name="cordova-blinkup-plugin">
```

Comment:
May need to rename BlinkUpPlugin.java. If so, maybe just rename the <name> (as above), but don't change the <feature>'s